client/src/pages/Login.js

```diff
@@ -0,0 +1,56 @@
+import logoBike from "./logoBike.svg";
+import Form from "react-bootstrap/Form";
+
+
+const Login = () => {
+	return (
+		<form>
+			<div className="Login-page">
+				<div className="flex-container">
+					<div className="img">
+						<div className="container-image">
+							<img src={logoBike} alt="bike logo" />
+						</div>
+
+						<h4>sign in</h4>
+					</div>
+					<div>
+						<>
+							<Form.Floating className="mb-3">
+								<Form.Control
+									id="floatingInputCustom"
+									type="email"
+									placeholder="name@example.com"
+								/>
+								<label htmlFor="floatingInputCustom">Email address</label>
+							</Form.Floating>
```

Comment:
```suggestion
			<Form.FloatingLabel
				controlId="email"
				label="Email address"
			>
				<Form.Control type="email" placeholder="Email address" />
			</Form.FloatingLabel>
```
The [React Bootstrap floating labels page](https://react-bootstrap.github.io/forms/floating-labels/#example) has an example with a `FloatingLabel` component with a `Form.Control` component inside it. We should use the same structure — can we update the code to use the same structure?

We also don't need to use a `label` element because we can pass in a `label` prop into `FloatingLabel` for the label.